exasollog-1.0.0-1.rockspec

```diff
@@ -0,0 +1,36 @@
+package = "exasollog"
+version = "1.0.0-1"
+
+source = {url = 'https://github.com/exasol/log-lua/releases/download/1.0.0/rock-exasollog-1.0.0.zip'}
+
+description = {
+    summary = "(Remote) logging for Exasol and other Lua contexts",
+    detailed = [[`exasollog` provides a logger for Lua that can log message to a remote log receiver listening on a TCP
+    port of a host reachable from the machine producing the log entries.
+    
+    It can also be used for simple console logging. If the connection cannot be established, the log automatically
+    falls back to STDOUT (in some cases this will at least let the messages appear in a local log file).
+     
+    `exasollog` depends on [CSJON](https://github.com/openresty/lua-cjson) (MIT license). Note that on an Exasol
+    cluster CJSON is pre-installed.
+
+    You can find the [user guide](https://github.com/exasol/log-lua/blob/master/doc/user-guide/user-guide.md) in the
```

Comment:
link is broken

Reply:
Fixed.

Reply:
Will be usable after first merge to master.

Reply:
:+1: